### PR TITLE
set upload_max_size and post_max_size to 128M

### PR DESCRIPTION
**Pull request by umjnega**

### CODE REVIEW

File: manifests/profile/www_lib/php.pp [hunk 1]
```diff
@@ -101,7 +101,8 @@
       'PHP/max_input_vars'          => '2000',
       'PHP/memory_limit'            => '256M',
       'PHP/error_reporting'         => 'E_ALL & ~E_DEPRECATED',
-      'PHP/upload_max_filesize'     => '64M',
+      'PHP/upload_max_filesize'     => '128M',
+      'PHP/post_max_size'	    => '128M',
```
Comment (botimer) [inline manifests/profile/www_lib/php.pp:105]: The linter is going to bark about alignment here.

**Review by botimer — CHANGES_REQUESTED** [review]
Once the linter is happy, this looks good to me.

File: manifests/profile/www_lib/php.pp (hunk 1 above)
Comment (umjnega) [reply manifests/profile/www_lib/php.pp]: ah, yeah tab instead of spaces.  Thanks.  Fixed.

**Comment by umjnega** [comment]
Fixed linter, it was a tab vs spaces.  My bad.

**Pull request merged**